style_editor.html

```diff
@@ -2503,7 +2498,7 @@
   b = mapcolor(b);
   var color = "rgb("+r+","+g+","+b+")";
   if (r == 0 && g == 0 && b == 0) color = "white";
-  return "<input type=button style='background:"+color+"' class=btn onclick='SetTo(\""+name+"\")' value='"+enc(name)+"'>\n";
+  return "<input type=button style='border: 1px solid black;padding:8px;color:black;background:"+color+"' class=btn onclick='SetTo(\""+name+"\")' value='"+enc(name)+"'>\n";
```

Comment:
In prep for Dark mode, hardcoding some styling on this dynamically created group of buttons keeps the colors and text from inverting. Also increased the buttons' size for a bit more color visibility per.